src/track_helper.cpp

```diff
@@ -180,7 +180,7 @@ TrackHelper::TrackHelper (std::string filename) :
 
 TrackHelper::~TrackHelper()
 {
-    if (!tracker)
+    if(tracker)
```

Comment:
You can simply remove the `if(tracker)`

Reply:
Should be fixed here : dbb5a9ad8227d426c79a983b36ea7fec86b957b0